addOns/automation/src/main/java/org/zaproxy/addon/automation/AutomationJob.java

```diff
@@ -188,6 +188,11 @@ public AutomationPlan getPlan() {
     }
 
     public void setChanged() {
```

Comment:
`resetAndSetChanged()` is clearer (and leave the "new" as `setChanged`)  ?

Reply:
That will introduce breaking changes across plugins...
Should we do that change anyway?

Reply:
Best not to introduce breaking changes :)

Reply:
There's no breaking changes, the method already exists and better name them properly. You do have to correct existing calls to call the new method.

Reply:
What I meant with breaking changes is:

The method `setChanged`  in automation addOn is called by several other AddOns.
We have to release them also, right? Indeed they will not "break" because you proposed to keep one `setChanged` method. But the fix will not be applied until they are released.

Reply:
That's fine, they are being released more often now.